spec/requests/api/v1/passwords/edit_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'GET api/v1/users/password/edit', type: :request do
+  let(:user) { create(:user, password: 'mypass123') }
+  let(:password_token) { user.send(:set_reset_password_token) }
```

Comment:
why not `user.set_reset_password_token` ?

Reply:
Because it is a protected method